cmd/spoc/main.go

```diff
@@ -17,28 +17,61 @@ limitations under the License.
 package main
 
 import (
+	"fmt"
 	"log"
 	"os"
+	"path/filepath"
 
 	"github.com/urfave/cli/v2"
 
 	"sigs.k8s.io/security-profiles-operator/cmd"
+	"sigs.k8s.io/security-profiles-operator/internal/pkg/cli/recorder"
 )
 
+const outputFileFlag = "output-file"
+
+var defaultOutputFile = filepath.Join(os.TempDir(), "profile.yaml")
+
 func main() {
 	app, _ := cmd.DefaultApp()
 	app.Usage = "Security Profiles Operator CLI"
 
 	app.Commands = append(app.Commands,
 		&cli.Command{
-			Name:    "record",
-			Aliases: []string{"r"},
-			Usage:   "run the recorder",
-			Action:  record,
+			Name:      "record",
+			Aliases:   []string{"r"},
+			Usage:     "run the recorder",
+			Action:    runRecord,
+			ArgsUsage: "COMMAND",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:        outputFileFlag,
+					Aliases:     []string{"o"},
+					Usage:       "the output file path for the recorded profile",
+					DefaultText: defaultOutputFile,
+					TakesFile:   true,
+				},
```

Comment:
We probably also want to add a parameter that specifies the type of recording we're going to do. It can default to `seccomp` which is what it's doing right now.

Reply:
Alright, added a type flag as well :+1: